### DISCUSSION TITLE
Can a cask be accepted even if GitHub repository not notable enough?

### DISCUSSION THREAD

**Discussion by viking1304** [Homebrew/discussions#3406] [Tap maintenance and Homebrew development] [answered]
I had the intention to submit a new cask for [TikDown](https://github.com/Tairraos/TikDown), but this is a new project that is not notable enough yet. I can help the author to maintain his own tap until the project met the required criteria, but I want to ask if you are willing to make an exception because this application is actively maintained and has unique options that do not exist in similar applications.

TikDown is an application similar to [4K Tokkit](https://www.4kdownload.com/products/product-tokkit/1), but unlike 4K Tokkit which only works with the international version of TikTok and requires a subscription for some options, TikDown is open source and works for both versions (international TikTok and Chinese douyin). For us who are using both TikTok versions, this app is more useful than 4K Tokkit.

If you think that this project is not acceptable after all, I will help @tairraos to maintain his own tap until the criteria are met.

**Comment by Rylan12** [comment, marked as answer]
It sounds unlikely that we would make an exception. We do have [a few criteria for exceptions](https://docs.brew.sh/Acceptable-Casks#exceptions-to-the-notability-threshold) to the notability rule for casks, but it doesn't look like those are met in this case.

I'd recommend just maintaining a separate tap until it is notable enough.

**Reply by viking1304** [reply]
Thank you. I will help @Tairraos to maintain his own tap until the criteria are met in that case.
